readme.md

```diff
@@ -56,6 +56,29 @@ Using the bundle
     is defined by the `rootPath` and `applicationContextPath` in your
     Dropwizard server configuration.
 
+5.  If your application uses [Feign](https://github.com/Netflix/feign) 
```

Comment:
prefacing somethign with if doesn't make it optional, it makes it conditional :-P